plugin-build/buildSrc/src/main/kotlin/io/sentry/android/gradle/internal/ASMifyTask.kt

```diff
@@ -45,7 +45,7 @@ abstract class ASMifyTask : Exec() {
     private val tmpDir: String get() = "${project.buildDir}/tmp/asmified"
 
     override fun exec() {
-        val asmJars = project.configurations.getByName("runtimeClasspath")
+        val asmJars = project.configurations.getByName("compileClasspath")
```

Comment:
This no longer worked, because we had made ASM dependencies `compileOnly`, so this had to change to compileClasspath